Runtime/RosMsgs/Vision/vision_msgs.cs

```diff
@@ -0,0 +1,117 @@
+using std_msgs;
+using std_msgs.msg;
+using geometry_msgs.msg;
+using sensor_msgs.msg;
+
+namespace vision_msgs
+{
+    namespace msg
+    {
+        public class BoundingBox3D : IRosMsg
+        {
+            string IRosMsg.GetRosType()
+            {
+                return "vision_msgs.msg.BoundingBox3D";
+            }
+
+            /// <summary>
+            /// The 3D position and orientation of the bounding box center.
+            /// </summary>
+            public Pose center;
+
+            /// <summary>
+            /// The size of the bounding box, in meters, surrounding the object's center pose.
+            /// </summary>
+            public Vector3 size;
+        }
+
+        public class ObjectHypothesis : IRosMsg
+        {
+            string IRosMsg.GetRosType()
+            {
+                return "vision_msgs.msg.ObjectHypothesis";
+            }
+
+            /// <summary>
+            /// The unique string ID of the object class.
+            /// </summary>
+            public string class_id;

```

Comment:
in ros2 this variable should be `class_id` for ros1 it should be `id`

[ROS1](http://docs.ros.org/en/noetic/api/vision_msgs/html/msg/ObjectHypothesis.html)
[ROS2](https://github.com/ros-perception/vision_msgs/blob/ros2/vision_msgs/msg/ObjectHypothesis.msg)